### ISSUE TITLE
Missing declaration file

### ISSUE THREAD

**Issue by eleallegue** [tiaanduplessis/react-native-phone-call#17]
I am reporting this, because I was about to use this library in my project and I have just done the `npm install --save react-native-phone-call`.
When I do `import call from 'react-native-phone-call` I get a visual studio code error that tells me this:
![image](https://user-images.githubusercontent.com/15983836/56284733-af9cb100-6115-11e9-8d33-e7dc74f992de.png)

I think you are missing what is there written in your index.js declaration. I will try to write that and give you a feed back, because the option of  `npm install @types/react-native-phone-call` does not work and returns:
`npm ERR! code E404`
`npm ERR! 404 Not Found: @types/react-native-phone-call@latest`

If you have any idea why this could be happening, I will be glad to hear it from you.

**Comment by asborisov** [comment]
```ts
declare module 'react-native-phone-call' {
  export type CallArguments = {
    number: string;
    prompt: boolean;
  }

  export default function call(args: CallArguments): Promise<void>;
}
```

**Comment by tiaanduplessis** [comment]
This should be available in v1.1.0

**Comment by chphoom** [comment]
I ran into this issue as well and had to manually add the index.d.ts file to fix it. is there a reason npm doesnt download this file automatically?
